pkg/controller/elasticsearch/driver/downscale_test.go

```diff
@@ -193,7 +194,10 @@ func TestHandleDownscale(t *testing.T) {
 	var actual appsv1.StatefulSetList
 	err := k8sClient.List(&actual)
 	require.NoError(t, err)
-	require.Equal(t, expectedAfterDownscale, actual.Items)
+	require.Equal(t, len(expectedAfterDownscale), len(actual.Items))
+	for i := range expectedAfterDownscale {
+		comparison.RequireEqual(t, &expectedAfterDownscale[i], &actual.Items[i])
```

Comment:
Would `comparison.RequireEqualSlices()` be a nice additional function to have?

Reply:
Good idea, I was trying to think of a good way to make it generic and failed. Maybe having a separate slice func makes sense though since I'm just repeating the same logic everywhere.